### PR TITLE
Add FEWD values to readme

### PR DESCRIPTION
**Pull request by cfarm**
We came up with these as part of the Browser Support Working Group. Rather than keeping them buried in a Google Doc somewhere, I'd like to place them somewhere prominent for future reference.

I thought about replacing the existing "Core Values" since, a) I don't know where those came from and if current team had much input and b) these new values were created as a group by FEWDs, which I think might better represent our culture. But BEWDs had no input in the new values so that's the main reason I held back from replacing. Thoughts on that?

### CODE REVIEW

**Comment by Scotchester** [comment]
The original core values were written almost entirely by Bill Shelton, IIRC. I think they're still valuable as general development team values, and the ones being added are fairly front-end specific, so I'd keep both lists.

File: README.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ Capital Framework is our modular front end framework. New projects should start
 - **Willingness**: We are willing to help each other with anything that is good for the team
 - **Excellence**: We believe that excellence is defined by a commitment to and practice of constant learning and improvement
 
+## Our Front End Web Development Values
+
+1. End goals of the user should be what drive the front-end design.
+2. Pages, apps, and sites should be inclusive of all users.
+3. We want to continue to use cutting-edge technologies and methods to build modern applications, but we won’t allow that to preclude usability for all consumers.
```
Comment (Scotchester) [inline README.md]: Usability it not something that you can preclude. How about "… won't allow that to negatively impact usability …"?

File: README.md [hunk 2]
```diff
@@ -53,6 +53,15 @@ Capital Framework is our modular front end framework. New projects should start
...
+4. Testing in supported browsers should be as painless as we can make it.
+5. Websites should download quickly and not cost smartphone users much money.
```
Comment (Scotchester) [inline README.md]: Maybe replace "smartphone users" with "users on metered data connections".

File: README.md [hunk 3]
```diff
@@ -53,6 +53,15 @@ Capital Framework is our modular front end framework. New projects should start
 - **Willingness**: We are willing to help each other with anything that is good for the team
 - **Excellence**: We believe that excellence is defined by a commitment to and practice of constant learning and improvement
 
+## Our Front End Web Development Values
```
Comment (Scotchester) [inline README.md]: Trivial "Front-end web development values" (and the prior heading should also be sentence-cased).

File: README.md (hunk 2 above)
Comment (jimmynotjim) [reply README.md]: Good point, I'm often tethered for service, but using a laptop.

**Comment by cfarm** [comment]
Made tweaks as requested so I'm gonna 🚢

**Pull request merged**